### PR TITLE
(DOCSP-3832) Pushing broken build for debugging.

### PR DESCRIPTION
**Pull request by rob-mongodb**
Staged: https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/rob-admin/DOCSP-3832/reference/mongosqld.html#configuration-file

### CODE REVIEW

File: source/includes/options-mongosqld.yaml [hunk 1]
```diff
@@ -828,4 +828,26 @@ description: |
...
+     - Type
+     - Corresponds to
+
+   * - ``polymorphic_type_conversion_mode``
+     - string
+     - Determines how :binary:`~bin.mongosqld` evaluates document fields that are specified with multiple data types.
+
+   * - ``type_conversion_mode``
+     - string
+     - Specifies the semantics that :binary:`~bin.mongosqld` uses for type conversions, such as the CAST function.
```
Comment (jdestefano-mongo) [inline source/includes/options-mongosqld.yaml]: Add a supplemental link to the system variables page that describes this behavior a bit better:
https://docs.mongodb.com/bi-connector/current/reference/system-variables/

```
For more information on type conversion modes, see :ref:`system-environment-variables`.
```

File: source/reference/mongosqld.txt [hunk 2]
```diff
@@ -660,17 +656,38 @@ Process Management Options
...
+Set Parameter Options
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+   setParameter:
+     polymorphic_type_conversion_mode: <string>
+     type_conversion_mode: <string>
+
+.. include:: /includes/option/option-mongosqld-setParameter.rst
```
Comment (jdestefano-mongo) [inline source/reference/mongosqld.txt]: I think we're getting command line options and configuration file settings confused here.

`--setParameter` is not a command line option.  This ticket is only for the configuration file setting.
You'll need to do the following:

1. Make my changes suggested in "options-mongosqld.yaml" (supplemental link and using `:setting:`)
2. Move everything inside the `description: |` block in "options-mongosqld.yaml" to this line.
3. Undo all changes to the "options-mongosqld.yaml" file.

File: source/includes/options-mongosqld.yaml [hunk 3]
```diff
@@ -828,4 +828,26 @@ description: |
...
+description: |
+
+  .. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Corresponds to
+
+   * - ``polymorphic_type_conversion_mode``
```
Comment (jdestefano-mongo) [inline source/includes/options-mongosqld.yaml]: Change to 
```
:setting:`setParameter.polymorphic_type_conversion_mode`
```

File: source/includes/options-mongosqld.yaml [hunk 4]
```diff
@@ -828,4 +828,26 @@ description: |
...
+     - string
+     - Determines how :binary:`~bin.mongosqld` evaluates document fields that are specified with multiple data types.
+
+   * - ``type_conversion_mode``
```
Comment (jdestefano-mongo) [inline source/includes/options-mongosqld.yaml]: Change to:
```
:setting:`setParameter.type_conversion_mode`
```

File: source/reference/mongosqld.txt [hunk 5]
```diff
@@ -276,6 +276,58 @@ You may configure :binary:`~bin.mongosqld` using a `YAML <http://yaml.org/>`_
...
+     - :option:`--sampleNamespaces <mongosqld --sampleNamespaces>`
+
+   * - .. setting:: schema.sample.refreshIntervalSecs
+     - integer
+     - :option:`--sampleRefreshIntervalSecs <mongosqld --sampleRefreshIntervalSecs>`
+
+   * - .. setting:: schema.sample.uuidSubtype3Encoding
+     - string
+     - :option:`--uuidSubtype3Encoding <mongosqld --uuidSubtype3Encoding>`
+
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/reference/mongosqld.txt]: I alphabetized the option groups to help the reader.  Scrambled the order up again after reaching your comment below. 
Separately, does this new directive mean that we no longer "look at the whole page" to fix other low-hanging fruit when addressing a ticket?  Our processes have changed so often that I can't keep current.

**Comment by rob-mongodb** [comment]
@jdestefano-mongo , updated and ready for another look!

File: source/reference/mongosqld.txt (hunk 2 above)
Comment (rob-mongodb) [reply source/reference/mongosqld.txt]: Fixed.

File: source/includes/options-mongosqld.yaml (hunk 3 above)
Comment (rob-mongodb) [reply source/includes/options-mongosqld.yaml]: Per your request, I removed all of the content related to this pr from this file.

File: source/includes/options-mongosqld.yaml (hunk 4 above)
Comment (rob-mongodb) [reply source/includes/options-mongosqld.yaml]: See above.

**Comment by jdestefano-mongo** [comment]
Picking this up under https://github.com/mongodb/docs-bi-connector/pull/227

**Pull request closed without merging**